lib/lure/vcs/git.go

```diff
@@ -36,7 +36,7 @@ func NewGit(auth Authentication, source string, to string, basePath string) (Git
 
 func (gitRepo GitRepo) SanitizeBranchName(branchName string) string {
 	//TODO: https://wincent.com/wiki/Legal_Git_branch_names
-	reg, _ := regexp.Compile("[^a-zA-Z0-9_-]+")
+	reg, _ := regexp.Compile("[^a-zA-Z0-9/_-]+")
```

Comment:
technically the branch cannot end with a /, so I would propose this regex instead `[a-zA-Z0-9/_-]*[a-zA-Z0-9_-]+`

Reply:
Also the ^ is an illegal character from what I can see :o so it should be removed as well

Reply:
🤦 thank you David, I forgot that within a character class the ^ counted as a negate operator

Reply:
we might still want to cover the case were the `/` is at the end, so would work in your opinion? `[^a-zA-Z0-9/_-]*[^a-zA-Z0-9/_-]` ?

Reply:
i think it'll require more than just changing the regex to support this since we're using `ReplaceAllString`

Reply:
bleh, ok nvm, lets leave it like that and create an issue to avoid the cornercase.